release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -438,19 +438,113 @@ With this update, the default credentials request for AWS has been modified to a
+
+For more information, see ???.
+
+[id="ocp-4-13-storage-vpshere-top-support-multi-datacenters"]
+==== VMware vSphere CSI topology support for multiple datacenters is generally available
+{product-title} 4.12 introduced the ability to deploy {product-title} for vSphere on different zones and regions, which allows you to deploy over multiple compute clusters, thus helping to avoid a single point of failure. {product-title} 4.13 introduces support for deploying over multiple datacenters and to set up the topology using failure domains created during installation or post-installation.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere-top-aware_persistent-storage-csi-vsphere[vSphere CSI topology].
+
+[id="ocp-4-13-storage-multiple-default-sc"]
+==== Creating more than one default storage class is generally available
+{product-title} 4.13 allows you create more than one default storage class. This feature makes it easier to change the default storage class because you can create a second storage class defined as the default. You then temporarily have two default storage classes before removing default status from the previous default storage class. While it is acceptable to have multiple default storage classes for a short time, you should ensure that eventually only one default storage class exists.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc[Managing the default storage class].
+
+[id="ocp-4-13-storage-managing-default-sc"]
+==== Managing the default storage class is generally available 
+{product-title} 4.13 introduces the `spec.storageClassState` field in the `ClusterCSIDriver` object, which allows you to manage the default storage class generated by {product-title} to accomplish several different objectives:
+
+* When you have other preferred storage classes, preventing the storage operator from re-creating the initial default storage class.
+
+* Renaming, or otherwise changing, the default storage class
+
+* Enforcing static provisioning by disabling dynamic provisioning.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc[Managing the default storage class].
+
+[id="ocp-4-13-storage-retroactive-default-sc-assignment"]
+==== Retroactive default StorageClass assignment (Technology Preview)
+Previously, if there was no default storage class, persistent volumes claims (PVCs) that were created that requested the default storage class remained stranded in the pending state indefinitely. {product-title} can now retroactively assign a default storage class to these PVCs, so that they do not remain in the pending state. With this feature enabled, after a default storage class is created, or one of the existing storage classes is declared the default, these previously stranded PVCs are assigned the default storage class.
```

Comment:
As for the PVCs being stranded indefinitely - this statement is not entirely true. Users had the option to delete and recreate the PVC, which would make it bind correctly. 

Otherwise this feature description looks good to me.

Reply:
@RomanBednar corrected.